saleor/graphql/warehouse/schema.py

```diff
@@ -0,0 +1,47 @@
+import graphene
+import graphene_django_optimizer as gql_optimizer
+from graphql_jwt.decorators import login_required
+
+from saleor.graphql.core.fields import PrefetchingConnectionField
+from saleor.graphql.decorators import permission_required
+from saleor.graphql.warehouse.mutations import (
+    WarehouseCreate,
+    WarehouseDelete,
+    WarehouseUpdate,
+)
+from saleor.graphql.warehouse.types import Warehouse
+from saleor.warehouse import models
+
+
+class WarehouseQueries(graphene.ObjectType):
+    warehouse = graphene.Field(
+        Warehouse,
+        description="Look up an order by ID.",
+        id=graphene.Argument(
+            graphene.ID, description="ID of an warehouse", required=True
+        ),
+    )
+    warehouses = PrefetchingConnectionField(
+        Warehouse, description="List of warehouses."
+    )
+
+    @login_required
```

Comment:
Do we need this? I assume that `permission_required` already requires logged in users.